docs/create-custom-image-using-existing-docker-image.md

```diff
@@ -171,18 +221,26 @@ docker push customrepository/custom-alfresco-repository:customTag
 ### Apply AMPs to the latest Share image (at this moment).
 FROM alfresco/alfresco-share:6.0.0-rc2
 
-### Copy the keystore build context to the appropriate location for your application server
-COPY keystore /usr/local/tomcat/keystore
+# Need these viariables to give permissions to tomcat (refer Dockerfile below)
```

Comment:
Please correct to:
"Default user and group are used to setup permissions for Tomcat process, see parent Dockerfile: Alfresco/acs-community-packaging/docker-alfresco/Dockerfile